sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLViewSuite.scala

```diff
@@ -95,15 +155,38 @@ class SQLViewSuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
       e = intercept[AnalysisException] {
         sql(s"""LOAD DATA LOCAL INPATH "$testData" INTO TABLE $viewName""")
       }.getMessage
-      assert(e.contains(s"Target table in LOAD DATA cannot be temporary: `$viewName`"))
+      assert(e.contains(s"Operation not allowed: LOAD DATA on temporary tables: `$viewName`"))
 
       e = intercept[AnalysisException] {
         sql(s"TRUNCATE TABLE $viewName")
       }.getMessage
       assert(e.contains(s"Operation not allowed: TRUNCATE TABLE on temporary tables: `$viewName`"))
+
+      e = intercept[AnalysisException] {
+        sql(s"SHOW CREATE TABLE $viewName")
+      }.getMessage
+      assert(e.contains(
+        s"Operation not allowed: SHOW CREATE TABLE on temporary tables: `$viewName`"))
+
+      e = intercept[AnalysisException] {
+        sql(s"SHOW PARTITIONS $viewName")
+      }.getMessage
+      assert(e.contains(s"Operation not allowed: SHOW PARTITIONS on temporary tables: `$viewName`"))
+
+      intercept[NoSuchTableException] {
+        sql(s"ANALYZE TABLE $viewName COMPUTE STATISTICS")
+      }
     }
   }
 
+  private def createTempView(viewName: String): Unit = {
```

Comment:
hmm, do we really need this method? I think it's just a one line code, e.g. `spark.range(10).createTempView(...)`